src/renderers/shaders/ShaderChunk/tonemapping_pars_fragment.glsl.js

```diff
@@ -22,17 +21,17 @@ vec3 ReinhardToneMapping( vec3 color ) {
 
 }
 
-// source: http://filmicgames.com/archives/75
+// source: http://filmicworlds.com/blog/filmic-tonemapping-operators/
 #define Uncharted2Helper( x ) max( ( ( x * ( 0.15 * x + 0.10 * 0.50 ) + 0.20 * 0.02 ) / ( x * ( 0.15 * x + 0.50 ) + 0.20 * 0.30 ) ) - 0.02 / 0.30, vec3( 0.0 ) )
 vec3 Uncharted2ToneMapping( vec3 color ) {
 
 	// John Hable's filmic operator from Uncharted 2 video game
 	color *= toneMappingExposure;
-	return saturate( Uncharted2Helper( color ) / Uncharted2Helper( vec3( toneMappingWhitePoint ) ) );
+	return saturate( Uncharted2Helper( color ) / Uncharted2Helper( vec3( 1.0 ) ) );
```

Comment:
I am not sure 1.0 is an appropriate value here. It could be greater, based on the use case. In fact, it should be RGB-valued.

I would remove `Uncharted` completely if you are going to do this.

Reply:
`1.0` is what was being passed before:
https://github.com/mrdoob/three.js/blob/master/src/renderers/WebGLRenderer.js#L112

Wouldn't mind removing `Uncharted` but I by removing this we'll know how many people are using it.

Reply:
>1.0 is what was being passed before:

That was a default value. Users could change it.

Given this PR, I would remove `Uncharted`.

Reply:
Any other one you would remove?

Reply:
Well, ultimately, we will need to support high-def monitors, and I anticipate we will be supporting alternate ACES Output Device Transforms. So there will be additional ACES transforms added at some point.

I am not sure of the benefit of the others, but maybe someone else has a strong opinion on that.

Currently, there is little harm in retaining them for now -- except for `Uncharted`.

Reply:
Sounds good! Yes, I'll remove `Uncharted` then.

Reply:
Thanks @WestLangley for your answer and the update on `ACESFilmicToneMapping`. Maybe the future version will be suitable for us.

@mrdoob Of course! I'll send it over to you in the morning (CEST).

Reply:
Thanks! If the new `ACESFilmicToneMapping` doesn't work for you, we'll have to figure out an API to add custom tonemappings.

Reply:
There you go: #19707

Reply:
Thank you so much for implementing this!

Reply:
Thank you for sharing the files/use case!